scripts/install.sh

```diff
@@ -68,43 +68,84 @@ trap install_success EXIT
 # Everything from this point onwards is optional.
 
 configure_systemd() {
-    if ! id ollama >/dev/null 2>&1; then
-        status "Creating ollama user..."
-        $SUDO useradd -r -s /bin/false -m -d /usr/share/ollama ollama
-    fi
-
     status "Creating ollama systemd service..."
-    cat <<EOF | $SUDO tee /etc/systemd/system/ollama.service >/dev/null
+    HOME=$(eval echo ~$(whoami))
+    # create the directories the service will need
+    mkdir -p $HOME/.config/systemd/user/
+    mkdir -p $HOME/.ollama/logs/
+    # add the service for the current user
+    cat <<EOF | $SUDO tee $HOME/.config/systemd/user/ollama.service >/dev/null
 [Unit]
-Description=Ollama Service
-After=network-online.target
+Description=Ollama
 
 [Service]
 ExecStart=$BINDIR/ollama serve
-User=ollama
-Group=ollama
 Restart=always
 RestartSec=3
-Environment="HOME=/usr/share/ollama"
+Environment="HOME=$HOME"
+StandardOutput=file:$HOME/.ollama/logs/server.log 
+StandardError=file:$HOME/.ollama/logs/server.log 
 Environment="PATH=$PATH"
 
 [Install]
 WantedBy=default.target
 EOF
-    SYSTEMCTL_RUNNING="$(systemctl is-system-running || true)"
+
+    SYSTEMCTL_RUNNING="$(systemctl --user is-system-running || true)"
     case $SYSTEMCTL_RUNNING in
         running|degraded)
             status "Enabling and starting ollama service..."
-            $SUDO systemctl daemon-reload
-            $SUDO systemctl enable ollama
+            systemctl --user daemon-reload
+            systemctl --user enable ollama
 
-            start_service() { $SUDO systemctl restart ollama; }
+            start_service() { systemctl --user restart ollama; }
             trap start_service EXIT
             ;;
     esac
 }
 
-if available systemctl; then
+migrate_systemd() {
+    # Check if the ollama service exists and is running
+    if systemctl is-active --quiet ollama; then
+        # Extract the User value from the service configuration
+        SERVICE_USER=$(systemctl show -p User --value ollama)
+
+        # If the service is running as the "ollama" user, stop it, copy content to the user's home, and remove it
+        if [ "$SERVICE_USER" = "ollama" ]; then
+            status "Ollama service is currently running as 'ollama' user. Migrating..."
```

Comment:
We could humanize this a bit since not many folks may not have looked into the details of the `ollama` user running.

```suggestion
            status "Detected a previous install of Ollama. Ollama now runs as your user. Migrating models... to ~/.ollama/models"
```